examples/clone.py

```diff
@@ -1,18 +1,37 @@
-#!/usr/bin/python
-# This trivial script demonstrates how to clone a remote repository.
-#
-# Example usage:
-#  python examples/clone.py git://github.com/jelmer/dulwich dulwich-clone
+"""Clone.
+
+This trivial script demonstrates how to clone a remote repository.
+
+Example usage:
+    2. python examples/clone.py git://github.com/jelmer/dulwich.git dulwich
+    1. python examples/clone.py git://github.com/jelmer/dulwich.git
```

Comment:
The numbering seems to be in the wrong order?

Reply:
Ok